apps/frontend/src/context/ActiveWorkoutContext.tsx

```diff
@@ -241,14 +239,12 @@ export const ActiveWorkoutContextProvider = ({
     });
   };
 
-  const increaseElapsedTime = (millis: number) => {
+  const increaseElapsedTime = React.useCallback((millis: number) => {
     dispatchActiveWorkoutAction({
       type: 'INCREASE_PART_ELAPSED_TIME',
       millis,
-      setResistance,
-      activeFtp,
```

Comment:
Figured that these were not being used anyways – might have affected the rendering i guess 🤷